docs/config.yaml

```
@@ -49,6 +53,13 @@ streams:
       additional_registry_repos:
         oscontainer_old: registry.ci.openshift.org/rhcos-devel/machine-os-oci-content
         legacy_oscontainer_old: registry.ci.openshift.org/rhcos-devel/machine-os-content
+      # OPTIONAL: override default coreos-assembler image to build with
+      cosa_img: quay.io/coreos-assembler/coreos-assembler:stable
+      # OPTIONAL/TEMPORARY: Override default coreos-assembler image to use for
+      # the controller pod in the build-arch job. This is only useful when
+      # building with a cosa too old to support `cosa remote`, which should only
```

I think it's `remote-session`:

```suggestion
      # building with a cosa too old to support `cosa remote-session`, which should only
```